### DISCUSSION TITLE
How to delete all accounts?

### DISCUSSION THREAD

**Discussion by gerkas** [Authenticator-Extension/Authenticator#1180] [General]
Hello all, I was wondering if when having several accounts created, is there a shortcut or some option to delete all accounts in one shot not one by one?

**Comment by Sneezry** [comment]
Right click the extension icon, then select Options, reset the extension. All your data will be cleared, including accounts and extension settings. You cannot undo this operation.

**Reply by gerkas** [reply]
When I do that it starts to move, bouncing very quickly and I can't select the reset button, I don't know what's happening. I have version 6.3.6


https://github.com/Authenticator-Extension/Authenticator/assets/133447152/5492c294-781c-4dbd-a7c1-578bd2c1b55e

**Reply by temuraru** [reply]
You should click the checkbox, then click the button bellow, while trying to ignore the bouncing. It worked for me, after some frustrating moments.. :)
